openlcs/openlcs/settings.py

```diff
@@ -103,6 +103,23 @@
     }
 }
 
+# The LOCATION could be modified to 'redis://127.0.0.1:6379/1' in local env
+CACHES = {
+    'default': {
+        'BACKEND': 'django_redis.cache.RedisCache',
+        'LOCATION': 'redis://redis:6379/1',
```

Comment:
I think we also need add it in the `settings.py`,  such as 
```
REDIS_CACHES_LOCATION = os.environ.get('REDIS_CACHES_LOCATION',
                                                                             'redis://localhost:6379/1')
```
so that we can use it on each envs or local env